__tests__/app.test.js

```diff
@@ -83,3 +83,25 @@ describe("GET /api/articles/:articleId", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("Should return status 200 and an array including all articles in the right format", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
```

Comment:
This is a good start to testing this endpoint, but there are a couple of key things to be added still.

If given an empty array your current test would pass, which is something we wouldnt want to happen as an empty array is not an array of articles, making it a false positive. In order to prevent this we need to ensure that the array contains something before entering your forEach.

Another key part of this endpoint is ensuring that the articles come back sorted by the created_at in descending order. I would recommend using the jest-sorted library to do this.